invenio_sipstore/models.py

```diff
@@ -155,6 +175,31 @@ def validate_key(self, filepath, filepath_):
     """Relation to the SIP along which given file was submitted."""
 
 
+class SIPMetadata(db.Model, Timestamp):
```

Comment:
I didn't change the content here, just added a way to be able to store multiple metadata. @lnielsen may have an answer?